### PR TITLE
feat: Add ability to paste env files

### PR DESCRIPTION
**Pull request by SaraVieira**
This PR adds a small textarea at the end of the env files page that allows a user to paste an env file.

CleanShot 2022-04-02 at 23 29 52

closes https://github.com/coollabsio/coolify/issues/263

### CODE REVIEW

**Comment by SaraVieira** [comment]
cc @andrasbacsai

File: src/routes/applications/[id]/logs/index.svelte [hunk 1]
```diff
@@ -105,7 +105,7 @@
 				</button>
 			</div>
 			<div
-				class="font-mono leading-6 text-left text-md tracking-tighter rounded bg-coolgray-200 py-5 px-6 whitespace-pre-wrap break-words overflow-auto max-h-[80vh] -mt-12 overflow-y-scroll scrollbar-w-1 scrollbar-thumb-coollabs scrollbar-track-coolgray-200"
```
Comment (SaraVieira) [inline src/routes/applications/[id]/logs/index.svelte:108]: This makes the applicaton logs always be fullwidth

**Comment by andrasbacsai** [comment]
Thank you! 💜

**Comment by jesperordrup** [comment]
Its not possible to flip the visiblity. 

#263

**Comment by andrasbacsai** [comment]
Ah, yeah, I totally missed it. A quick fix is already on its way.

**Comment by andrasbacsai** [comment]
Fixed in 2.3.1

**Pull request merged**